{"name": "test_login_none_existing_user", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLoginNegative' object has no attribute 'driver'", "trace": "self = <test_login_negative.TestLoginNegative object at 0x0000018C657CDB70>\n\n    @pytest.mark.tcid12\n    def test_login_none_existing_user(self):\n>       my_account = MyAccountSignOut(self.driver)\nE       AttributeError: 'TestLoginNegative' object has no attribute 'driver'\n\nqatest\\tests\\my_account\\test_login_negative.py:10: AttributeError"}, "start": 1648839716457, "stop": 1648839716457, "uuid": "bca0099d-6129-46ca-bf19-178409455e76", "historyId": "3b923004665eea39b536ed74517924ae", "testCaseId": "21a4a96dc547e68d9b797278d1c8b6ae", "fullName": "qatest.tests.my_account.test_login_negative.TestLoginNegative#test_login_none_existing_user", "labels": [{"name": "tag", "value": "tcid12"}, {"name": "parentSuite", "value": "qatest.tests.my_account"}, {"name": "suite", "value": "test_login_negative"}, {"name": "subSuite", "value": "TestLoginNegative"}, {"name": "host", "value": "DESKTOP-G3LLG73"}, {"name": "thread", "value": "7596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "qatest.tests.my_account.test_login_negative"}]}